tests/array/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -31,7 +31,7 @@ end
 strings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 def strings
-  ['    a', '    b', '    c', '    d']
+  [    a,     b,     c,     d]
```

Comment:
Context: https://github.com/iamsolankiamit/prettier-ruby/pull/21/files#r192564649

This is actually closer to what we would want now, even though it is definitely invalid code haha